sklearn/hpsearch/README.md

```diff
@@ -0,0 +1,34 @@
+# Parallelized Hyperparameter Search with Google Container Engine
+
+## Introduction
+
+This sample package helps you run `scikit-learn`'s `GridSearchCV` and `RandomizedSearchCV` on [Google Container Engine](https://cloud.google.com/container-engine/).
+
+
+## Requirements
+
+You will need a Google Cloud Platform project which has the following products enabled:
```

Comment:
Link to documentation about enabling APIs: https://support.google.com/cloud/answer/6158841?hl=en ?

Reply:
done